src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populationArray = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const countriesCount = populationArray.length;
+let total = 0;
+
+populationArray.forEach((populationString) => {
+  const populationNum = parseFloat(
+    populationString.textContent.replace(/,/g, ''),
+  );
+
+  total += populationNum;
+});
+
+const average = Math.floor(total / countriesCount);
```

Comment:
The calculation of the average population uses `Math.floor`, which rounds down the result to the nearest integer. If you want a more accurate average, consider using `Math.round` instead. However, if the task specifically requires rounding down, then this is correct.